core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/AbstractWorkflow.java

```diff
@@ -0,0 +1,48 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Objects'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)